Changes.md

```diff
@@ -16,7 +16,9 @@ Improvements
 Fixes
 -----
 
-- Box/Reference : Fixed export and referencing of TransformPlugs with modified default values (#3946).
+- Box/Reference :
+  - Fixed export and referencing of TransformPlugs with modified default values (#3946).
+  - Fixed export and referencing of CompoundDataPlugs with modified default values (#3907).
```

Comment:
![image](https://user-images.githubusercontent.com/896779/99650749-b7d57480-2a4d-11eb-9481-073bf9a4f5f3.png)

#3907 still seems to be broken:

 - Paste box from ticket
 - Export as reference
 - Import into Reference node

(Also broken if recreated from scratch).

In addition. You are presented with the 'Plugs not at default values' modal, but as they're NVPs with editable names, there is no visual indication that any plugs aren't at their default values. Maybe we could include a list of plugs in the dialog (or summary if there are too many)?

Reply:
See https://github.com/GafferHQ/gaffer/issues/3907#issuecomment-721077719. There are two issues here. I've fixed the first and am proposing we spin off a second ticket for the second.

> Maybe we could include a list of plugs in the dialog (or summary if there are too many)?

The message is already a bit wordy. Perhaps this could go in a collapsible details pane?

Reply:
Ah right, got you. Added new issue #4020 as this one will be closed on merge.

Reply:
Thanks, I've added some comments on that, assigned it to me, and put it in "Up Next". I'm hopeful that's it's a reasonably easy one.

Reply:
I have this fixed and will make a PR for it once I finally get this one over the line.